audit/projects/k8s-artifacts-prod/buckets/k8s-artifacts-cni/metadata.txt

```diff
@@ -12,8 +12,8 @@ gs://k8s-artifacts-cni/ :
 	Labels:				None
 	Default KMS key:		None
 	Time created:			Tue, 28 Jan 2020 17:29:02 GMT
-	Time updated:			Mon, 24 May 2021 17:36:41 GMT
-	Metageneration:			68
+	Time updated:			Thu, 10 Jun 2021 20:21:42 GMT
+	Metageneration:			69
```

Comment:
... so I suspect these bucket changes are from me running `./infra/gcp/ensure-artifacts-prod.sh k8s-artifacts-prod` but I never saw anything change while doing so

I haven't messed with that script much to make it more idempotent / change-only-if-needed, so it's possible this will get bumped every time that script is run